src/main/java/core/basesyntax/model/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random randomizer = new Random();
+
+    public String getRandomColor() {
```

Comment:
This methoid should return color, in our program color is enum, so please return propriate value here